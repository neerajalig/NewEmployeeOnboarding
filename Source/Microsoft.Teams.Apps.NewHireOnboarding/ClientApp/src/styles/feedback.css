
.dialog-container-view-feedback {
    background: transparent !important;
    padding: 0rem !important;
}

.input-fields-margin-between-add-post {
    margin-top: 1rem;
}
.margin-top {
    margin-top:0.7rem;
}
.margin-right {
    margin-right: 1rem;
}
.margin-bottom {
    margin-bottom: 1rem;
}
.margin-left {
    margin-left: inherit!important;
}
.input-label-space-between {
    margin-top: 0.3rem;
}

.dateContainer {
    width: 55% !important;
}

.dropdown-button {
    width: 50% !important;
}

.feedback-filter-section {
    margin-bottom: 3rem;
}


.feedback-message {
    text-align: center;
    width: 80%;
    height: 10rem;
    position: absolute;
    top: 4rem;
    bottom: 0;
    left: 0;
    right: 0;
    margin: auto;
    font-size: 1.5rem;
}

.feedback-excel-section {
    width: auto!important;
    overflow: auto;
}

/*Buttons width*/
.id {
    max-width: 15rem !important;
}

.margin-button-top {
    margin-top: 1rem;
}

@media only screen and (min-width: 651px) and (max-width: 700px) {
    .margin-button-top {
        margin-top:1rem;
    }
}
