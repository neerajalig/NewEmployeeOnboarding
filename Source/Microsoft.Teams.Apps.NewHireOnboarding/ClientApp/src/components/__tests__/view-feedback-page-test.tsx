import * as React from "react";
import { Provider } from "@fluentui/react-northstar";
import FeedbackPage from "../view-feedback/view-feedback-page";

import 'bootstrap/dist/css/bootstrap.min.css';
import "../../styles/feedback.css";
import { act } from "react-test-renderer";
import { render, unmountComponentAtNode } from "react-dom";
import pretty from 'pretty';

jest.mock('../../api/view-feedback-api');
jest.mock('react-i18next', () => ({
    useTranslation: () => ({
        t: (key: any) => key,
        i18n: { changeLanguage: jest.fn() }
    }),
    withTranslation: () => (Component: any) => {
        Component.defaultProps = { ...Component.defaultProps, t: (key: any) => key == "months" ? "January,February,March,April,May,June,July,August,September,October,November,December" : key };
        return Component;
    }
}));
jest.mock('react-spreadsheet', () => ({
    
}));


let container: any = null;
beforeEach(() => {
    // setup a DOM element as a render target
    container = document.createElement("div");
    // container *must* be attached to document so events work correctly.
    document.body.appendChild(container);
    
    act(() => {
        render(<Provider><FeedbackPage /></Provider>, container);
    });
});
afterEach(() => {
    // cleanup on exiting
    unmountComponentAtNode(container);
    container.remove();
    container = null;
});

describe('ViewFeedback', () => {
    it('renders snapshots', () => {
       expect(pretty(container.innerHTML)).toMatchSnapshot();
    });
});

