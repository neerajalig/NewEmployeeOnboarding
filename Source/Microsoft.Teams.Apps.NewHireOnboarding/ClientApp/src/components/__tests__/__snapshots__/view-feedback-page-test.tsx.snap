// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewFeedback renders snapshots 1`] = `""`;
